packages/nextjs-certificates-example/README.md

```diff
@@ -1,37 +1,97 @@
 
 
-# Mentaport NextJs SDK Example
+![logo](https://certificates.mentaport.com/_next/static/media/logo-menta.a1f4e18c.png)
```

Comment:
Logo being used is white and not visible. Lets change it to (https://cdn.prod.website-files.com/65da8b02938c83b4e797eb0f/65dbd2cbaad00145a343aefe_Title_1.png) or remove

Reply:
fixed